CVE-2016-1897/cvex.yml

```diff
@@ -0,0 +1,10 @@
+blueprint: ubuntu2204-ubuntu2204
+ubuntu1:
+  playbook: ubuntu1.yml
+  command:
+    - "php -S 0.0.0.0:9999 -f /var/www/html"
+ubuntu2:
+  playbook: ubuntu2.yml
+  command:
+    - "/var/www/generate_data.sh %ubuntu1%"
```

Comment:
If I convert "php -S 0.0.0.0:9999 -f /var/www/html" to "php -S 0.0.0.0:9999 -f /var/www/html&", and then let this command run, I get this:

> 2024-11-18 12:36:28,598 - CRITICAL - [main] Command failed: <UnexpectedExit: cmd='/var/www/generate_data.sh 192.168.56.3' exited=126>